.gitignore

```diff
@@ -1,3 +1,5 @@
+# Custom build script
+build.ps1
```

Comment:
I have 
`docker run --entrypoint cmd --rm -v $HOME\Desktop\output:C:\output -v C:\go_workspace\src\github.com\codilime\contrail-windows-docker:C:\go\src\github.com\codilime\contrail-windows-docker driverbuildimage /c powershell C:\generate.ps`
in here but other peoples' paths may be different, hence .gitignore